starfish/experiment/experiment.py

```diff
@@ -89,11 +121,63 @@ def name(self) -> str:
     def image_types(self) -> Set[str]:
         return set(self._images.keys())
 
-    def get_image(self, item: str, crop_parameters: Optional[CropParameters]=None) -> ImageStack:
-        return ImageStack.from_tileset(self._images[item], crop_parameters)
+    def show_aligned_image_groups(self) -> None:
+        """
+        Describe the aligned subgroups for each Tileset in this FOV
+
+        ex.
+        {'nuclei': ' Group 0:  <starfish.ImageStack r={0}, ch={0}, z={0}, (y, x)=(190,270)>',
```

Comment:
Did we decide on no coordinates?  I can't remember. :(

Reply:
yea no coords since we're moving towards making the cli less granular